graphics/nxmu/nxmu_server.c

```diff
@@ -86,10 +86,10 @@ static inline void nxmu_connect(FAR struct nxmu_conn_s *conn)
    * client
    */
 
-  ret = nxmq_open(mqname, O_WRONLY, 0, NULL, &conn->swrmq);
-  if (ret < 0)
+  conn->swrmq = nxmq_open(mqname, O_WRONLY);
+  if (conn->swrmq == (mqd_t)-1)
```

Comment:
change to conn->swrmq < 0 since nxmq_open return the negative error code in the fail case

Reply:
OK